ts/sagas/wallet/pagopaApis.ts

```diff
@@ -661,8 +661,8 @@ export function* paymentIdPollingRequestHandler(
       typeof isPagoPATestEnabledSelector
     > = yield select<GlobalState>(isPagoPATestEnabledSelector);
 
-    const response: SagaCallReturnType<typeof getPaymentIdApi> = yield call(
-      getPaymentIdApi,
+    const response: SagaCallReturnType<typeof getPaymentIdApi.e2> = yield call(
+      getPaymentIdApi.e2,
```

Comment:
for clarity it would be worth destructuring the duple into temporary variables with proper names